### PR TITLE
Understanding EtherChannel  Technology

### PR DESCRIPTION
**Pull request by dentonya**
### Article checklist
**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker? We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by dentonya** [comment]
cc @BrianaNzivu

**Comment by BrianaNzivu** [comment]
> cc @BrianaNzivu

Sorry for the wait starting the review process ASAP.

Regards.

File: articles/etherchannel-technology/index.md [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+
+We have to configure the network to use one logical link instead of the two fastEthernet(Fa) links connecting the switches.
+We shall use 3 port-channels:
+- Port-channel 1-link connecting S1 and S2 (Fa0/1, Fa0/2 ).
+- Port-channel 2-link connecting S2 and S3 (Fa0/3, Fa0/4 )
+- Port-channel 3-link connecting S1 and S3 (Fa0/5, Fa0/6 )
+
+We shall use LACP on port-channels 1 and 2 and PAgp on Channel 3 through the following steps;
+
+#### Basic Switch Configurations
```
Comment (BrianaNzivu) [inline articles/etherchannel-technology/index.md:111]: Show your steps i.e 
Step One - Basic Switch Configurations.
Step Two - Port-channel 2 configurations using LACP mode passive and active.

**Review by BrianaNzivu — CHANGES_REQUESTED** [review]
Hey @dentonya . Good job! The article is well articulated and easy to understand. Good job once again. I have requested a few changes. Please make these changes as I continue with the grammar review. Also, let me know where I can send you your plagiarism report.

Regards.

**Comment by dentonya** [comment]
Thanks for the review...will do the necessary changes ...the report you can
send it to this email

**Comment by dentonya** [comment]
Thanks for the review @BrianaNzivu will do the necessary changes ..

File: articles/etherchannel-technology/index.md (hunk 1 above)
Comment (dentonya) [reply articles/etherchannel-technology/index.md]: hello @BrianaNzivu done with the changes

**Comment by dentonya** [comment]
@BrianaNzivu  done with the changes

**Comment by dentonya** [comment]
@BrianaNzivu  seen the changes and what about the plagiarism report??

**Comment by dentonya** [comment]
cc @hectorkambow @BrianaNzivu

**Comment by BrianaNzivu** [comment]
Hey @dentonya . I hope you are doing ok. Good job on the article, very informative. Though, I have noticed your article has a lot of grammar mistakes. I'd suggest you first cross-check your work using third-party apps like [Grammarly]("https://app.grammarly.com/") which at least will save some time when it comes to reviewing your article. Next, I'd suggest you research more on Technical writing just to improve your skills. Remember there is no limit to what we can learn. I know [Google]("https://developers.google.com/tech-writing") has one of the best technical writing courses. Lastly, do take time to go through all of the changes I have made. I have done them per section so that you can clearly see these changes.
Otherwise, good job.
Regards.

**Review by BrianaNzivu — APPROVED** [review]
Great. Thanks for the changes.

**Comment by dentonya** [comment]
Thanks for the review @BrianaNzivu ..will do all  the grammar checks...and improve in all the areas you have highlighted

**Comment by BrianaNzivu** [comment]
> Thanks for the review...will do the necessary changes ...the report you can send it to this email

Sent

**Comment by BrianaNzivu** [comment]
Hi. Seems like I can't send the plagiarism report through this channel.
Kindly send me your personal email.

**Comment by dentonya** [comment]
Ok denatonya@gmail.com

**Comment by dentonya** [comment]
@BrianaNzivu  denatonya@gmail.com

**Comment by dentonya** [comment]
@BrianaNzivu  done with the changes

**Comment by BrianaNzivu** [comment]
Tech review done 🚀

**Comment by dentonya** [comment]
cc @BrianaNzivu @hectorkambow

**Comment by dentonya** [comment]
Ok seen the changes 👍 @BrianaNzivu @hectorkambow

**Comment by BrianaNzivu** [comment]
Great. Just kindly be patient Hector will review your article since there
is normally a queue followed.

Regards.

**Comment by dentonya** [comment]
Ook @BrianaNzivu thanks for the reminder

**Comment by hectorkambow** [comment]
@dentonya  👋  Good afternoon 

Just started a final review on your article - upon running your article through our 3rd party plagiarism checker it raise a few red flags and the % was bit higher than we typically like to accept/publish.

(see pdf)

[Understanding EtherChannel Technology #1920.pdf](https://github.com/section-engineering-education/engineering-education/files/6329952/Understanding.EtherChannel.Technology.1920.pdf)


Also noticed that your PR has a merged conflict which will prevent us from pushing it to the main repo, one of the easiest way to resolve is to create a fresh/new PR with the edited files from here.

**Comment by hectorkambow** [comment]
image

**Comment by dentonya** [comment]
Thanks for the feedback ,will work on the requested changes @hectorkambow

**Comment by dentonya** [comment]
So that means I will have to delete this PR @hectorkambow

**Comment by hectorkambow** [comment]
#2085

**Comment by dentonya** [comment]
cc @hectorkambow why is the PR #2085 taking so long yet it was a duplicate

**Comment by hectorkambow** [comment]
@dentonya 
Understood - our apologies for the delay - once it went back into the queue it went back in line - along with our article cap - that is why. With the new month rolling in - I'll be sure to check it out asap.

**Comment by dentonya** [comment]
@hectorkambow ok thanks for the clarification

**Pull request closed without merging**